src/plugins/persistence/couch/CouchObjectProvider.js

```diff
@@ -441,84 +421,58 @@ class CouchObjectProvider {
 
             this.changesFeedSharedWorker.port.postMessage({
                 request: 'changes',
-                body,
                 url
             });
         }
     }
 
-    async fetchChanges(url, body) {
-        const controller = new AbortController();
-        const signal = controller.signal;
+    onEventError(error) {
+        console.error('Error on feed', error);
+        if (Object.keys(this.observers).length > 0) {
+            this.observeObjectChanges();
+        }
+    }
+
+    onEventMessage(event) {
+        const object = JSON.parse(event.data);
```

Comment:
need to check here for the object type. if it's not a SYNCHRONIZED_OBJECT, toss it.

Reply:
Agreed. Open MCT does not have general handling for asynchronous remote object changes yet. But will soon 😄